Grammar fixes (#1966)
hacks-guide/Guide_3DS: _pages/en_US/installing-boot9strap-(ntrboot).txt

1. Remove the magnet from your device
  + On old 2DS, you should instead disable the sleep mode switch
1. When prompted, input the key combo given on the top screen to install boot9strap

> 1. Once it has been completed, force your device to power off by holding down the power button

Reviewer: ```suggestion
1. Once it is complete, force your device to power off by holding down the power button
```

Reviewer: This was not added.
